v2/netconf/server/ssh/server.go

```diff
@@ -0,0 +1,95 @@
+package ssh
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// Server represents a test SSH Server
+type Server struct {
+	listener net.Listener
+	trace    *Trace
+}
+
+// Handler is the interface that is implemented to handle an SSH channel.
+type Handler interface {
+	// Handler is a function that handles i/o to/from an SSH channel
+	Handle(ch ssh.Channel)
+}
+
+// HandlerFactory is a test function that will deliver an Handler.
+type HandlerFactory func(conn *ssh.ServerConn) Handler
+
+// NewServer deflivers a new test SSH Server, with a custom channel handler.
```

Comment:
did you mean to say test in the comment?

Reply:
Removed (hangover from origin as the netconf test server)